### ISSUE TITLE
The same billing address and delivery address

### ISSUE THREAD

**Issue by juraj1000** [PrestaShop/PrestaShop#19729]
#### Describe the bug
If a customer uses the same billing address and delivery address then both addresses are used in an invoice or invoice templates (e.g. new order). It is sufficient to use only billing address in this case. Currently it is a little bit confusing for the customers because majority of them have the same billing and delivery addresses.

#### Expected behavior
Use only billing address if billing address and delivery address are the same.

#### Additional information
* PrestaShop version: 1.7.6.5
* PHP version: 7.1.13

**Comment by matks** [comment]
Hi @juraj1000 do you have examples of other systems that do only display 1 address ? Because for years I have been shopping online and I have always seen both addresses being displayed, even when they are identical.

I even wonder if it's not mandatory by the law

**Comment by Gipielle** [comment]
Right, @juraj1000 , in ps 1.5 we made a customization, now in ps 1.7 we use identical address, but will be better to have only one.

**Comment by Hlavtox** [comment]
@matks @florine2623 

Out of 4 last invoices I got, 3 of them have only 1 address.

![2B0C73B4-3359-4B69-AB54-055BD4EB7997](https://user-images.githubusercontent.com/6097524/84553002-f0d92280-ad12-11ea-97f4-1974ce910e63.jpeg)

![34CA9547-93DE-4385-8CA1-CD9E094CBCC4](https://user-images.githubusercontent.com/6097524/84553000-efa7f580-ad12-11ea-8bd4-07855ac6e98b.jpeg)
![380B263B-AB29-41DC-9880-9ED0E9E872DF](https://user-images.githubusercontent.com/6097524/84552989-eb7bd800-ad12-11ea-9ea0-fdd32292bb9e.jpeg)

**Comment by juraj1000** [comment]
Hi @matks,
I think the configuration for one or 2 addresses should solve this issue. Let the e-shop company choose which option is better for business.

**Comment by florine2623** [comment]
Hi @PrestaShop/prestashop-product-team ,

What do you think about letting the merchant choose if he needs to use a different billing and delivery address ?

Thanks!

**Comment by Gipielle** [comment]
> Hi @PrestaShop/prestashop-product-team ,
> 
> What do you think about letting the merchant choose if he needs to use a different billing and delivery address ?
> 
> Thanks!

If these are the same ...
